web/themes/joinup/templates/content/node--event--full.html.twig

```diff
@@ -10,25 +10,39 @@
 <article{{ attributes }}>
   <div class="page__content">
     {{ content.body }}
-    {{ content.field_event_spatial_coverage }}
-    {{ content.field_event_agenda }}
-    {{ content.field_attachment }}
+    <div class="fieldset fieldset--box">
```

Comment:
The requirement states `A grey line with the title References that contains:` not `A grey block with ...`

Reply:
I think that "A grey line" could be mistake in the description. All other fieldsets have grey box.

Reply:
Agreed.